src/modules/auth/auth.service.ts

```diff
@@ -12,7 +12,7 @@ import { randomBytes } from 'crypto';
 import * as bcrypt from 'bcrypt';
 
 const jwt = require('jsonwebtoken');
-
+const saltNumber = 8;
```

Comment:
great, but such constant should be uppercase, to show that its value never changed SALT_NUMBER

Reply:
ok